e2e/interchaintestv8/testvalues/values.go

```diff
@@ -28,8 +28,8 @@ const (
 	EnvKeyOperatorPrivateKey = "PRIVATE_KEY"
 	// EnvKeySp1Prover The prover type (local|network|mock).
 	EnvKeySp1Prover = "SP1_PROVER"
-	// EnvKeySp1PrivateKey Private key for the prover network.
-	EnvKeySp1PrivateKey = "SP1_PRIVATE_KEY"
+	// EnvKeyNetworkPrivateKey Private key for the sp1 prover network.
+	EnvKeyNetworkPrivateKey = "NETWORK_PRIVATE_KEY"
```

Comment:
I don't love this name. I assume the change came from the builtin env prover?

Reply:
Yep